srcpkgs/rsyslog/template

```diff
@@ -40,6 +40,10 @@ post_install() {
 	vconf ${FILESDIR}/rsyslog.conf
 }
 
+do_check() {
```

Comment:
why

Reply:
Package builds on all archs and works on x86_64. I am not interested on fixing the failing test suite or to document why it fails. I am interested in building and using it.

Reply:
you are not surprised me asking you `why` if you don't document your rationale for disabling this, are you?
there's a reason CI runs do_check now, so you are at least expected to investigate and document your findings.